miner/src/miner.rs

```diff
@@ -232,29 +252,48 @@ impl MinerService for Miner {
 		Vec<Result<TransactionImportResult, Error>>
 		where T: Fn(&Address) -> AccountDetails {
 		let mut transaction_queue = self.transaction_queue.lock().unwrap();
-		transaction_queue.add_all(transactions, fetch_account)
+		let is_local = false;
```

Comment:
Just to explain what `false` flag is about - you think it should be inline or changed to enum?